jsapp/js/account/security/email/emailSection.component.tsx

```diff
@@ -120,20 +136,24 @@ export default function EmailSection() {
   return (
     <section className={securityStyles.securitySection}>
       <div className={securityStyles.securitySectionTitle}>
-        <h2 className={securityStyles.securitySectionTitleText}>{t('Email address')}</h2>
+        <h2 className={securityStyles.securitySectionTitleText}>
+          {t('Email address')}
+        </h2>
       </div>
-
       <div className={cx(securityStyles.securitySectionBody, styles.body)}>
         {!session.isPending &&
-          session.isInitialLoadComplete &&
-          'email' in currentAccount && (
-            <TextBox
-              value={email.newEmail}
-              placeholder={t('Type new email address')}
-              onChange={onTextFieldChange.bind(onTextFieldChange)}
-              type='email'
-            />
-          )}
+        session.isInitialLoadComplete &&
```

Comment:
I think there was too much logic in this template even before adding another boolean check. But adding in userCanChangeEmail really pushes it over the edge for me. I'd say you can abstract these out as a single variable: 
```
!session.isPending &&
session.isInitialLoadComplete &&
'email' in currentAccount &&
```
While we're at it, do we actually need to check both isInitialLoadComplete and isPending? Maybe we do, but it smells redundant to me.

And those checks are actually duplicated in two places right next to each other (see lines 159-161). At a glance, I'd say we can combine these and then just nest the check for unverifiedEmail?.email.